packages/network/lib/uri.ts

```diff
@@ -64,33 +62,19 @@ const removeDefaultPort = function (urlToCheck) {
   return parsed
 }
 
-const addDefaultPort = function (urlToCheck) {
+export function addDefaultPort (urlToCheck) {
   const parsed = parseClone(urlToCheck)
 
   if (!parsed.port) {
     // unset host...
     // see above for reasoning
-    parsed.host = null
-    parsed.port = DEFAULT_PROTOCOL_PORTS[parsed.protocol]
+    parsed.host = undefined
+    parsed.port = DEFAULT_PROTOCOL_PORTS[parsed.protocol || 'http:']
```

Comment:
Do you have a test for this change in code? Doesn't seem like a direct port from the previous behavior.

Reply:
There were actually no unit tests for `uri` when I ported it :grimacing: This was a change necessitated by Typescript. I guess this does represent a behavior change, since it would make an invalid URL have a port of 80, so I changed it to model the old behavior (in a way that typescript likes):

https://github.com/cypress-io/cypress/blob/ab5282ffe1f315820f1b9a83d2c3f1af3d117001/packages/network/lib/uri.ts#L65-L81